src/addrman.h

```diff
@@ -691,6 +697,9 @@ class CAddrMan
     //! Holds addrs inserted into tried table that collide with existing entries. Test-before-evict discipline used to resolve these collisions.
     std::set<int> m_tried_collisions;
 
+    /** Perform consistency checks every m_consistency_check_ratio operations (if non-zero). */
+    const int32_t m_consistency_check_ratio;
```

Comment:
7883160 negative values would be meaningless and value is clamped in range 0...1000000; could be unsigned

<details><summary>diff</summary><p>

```diff
--- a/src/addrman.h
+++ b/src/addrman.h
@@ -27,7 +27,7 @@ 
 /** Default for -checkaddrman */
 /** Default for -checkaddrman */
-static constexpr int32_t DEFAULT_ADDRMAN_CONSISTENCY_CHECKS{0};
+static constexpr uint32_t DEFAULT_ADDRMAN_CONSISTENCY_CHECKS{0};
 
@@ -495,7 +495,7 @@ public: 
-    explicit CAddrMan(bool deterministic, int32_t consistency_check_ratio)
+    explicit CAddrMan(bool deterministic, uint32_t consistency_check_ratio)

@@ -704,7 +704,7 @@ private:
     /** Perform consistency checks every m_consistency_check_ratio operations (if non-zero). */
-    const int32_t m_consistency_check_ratio;
+    const uint32_t m_consistency_check_ratio;

--- a/src/init.cpp
+++ b/src/init.cpp
@@ -1165,7 +1165,7 @@ bool AppInitMain(NodeContext& node, interfaces::BlockAndHeaderTipInfo* tip_info)
 
     assert(!node.addrman);
-    auto check_addrman = std::clamp<int32_t>(args.GetArg("-checkaddrman", DEFAULT_ADDRMAN_CONSISTENCY_CHECKS), 0, 1000000);
+    auto check_addrman = std::clamp<uint32_t>(args.GetArg("-checkaddrman", DEFAULT_ADDRMAN_CONSISTENCY_CHECKS), 0, 1000000);
```
</p></details>

Reply:
Since we're already clamping the value, I don't think it matters.